upgrade/client.py

```diff
@@ -8,6 +7,16 @@
 from fabric.api import execute
 from fabric.api import run
 
+from upgrade.helpers.constants import settings
+from upgrade.helpers.constants.environment_config import CLIENT6_HOSTS
```

Comment:
Same as suggested above for jinja formatting and using directly with settings object.

Reply:
duplicate, replied at https://github.com/SatelliteQE/satellite6-upgrade/pull/467/files#r602082977